schemas/src/main/resources/profileschema/0.1/datahelix.profile.bnf

```diff
@@ -43,9 +43,9 @@ equalTo ::= '"' 'equalTo' '"' ',' '"' 'value' '"' ':' ( number | string | tempor
 temporal         ::= '{' '"' 'date' '"' ':' '"' temporalValue '"' '}'
 temporalValue    ::=temporalYear'-'temporalMonth '-'temporalDay 'T' temporalHour ':' temporalMinute ':' temporalSecond '.' temporalMillisec
 temporalYear     ::= digit digit digit digit
-temporalMonth    ::= ( '0' digit ) | ( '1' ( [0-2] ) )
-temporalDay      ::= ( [0-2] digit ) | ( '3' ( [0-1] ) )
-temporalHour     ::= ( [0-1] digit ) | ( '2' ( [0-3] ) )
+temporalMonth    ::= ( '0' [1-9] ) | ( '1' ( [0-2] ) )
+temporalDay      ::= ( '0' [1-9] ) | ( [1-2] digit ) | ( '3' ( [0-1] ) )
```

Comment:
The simplified versions of these (when run through https://www.bottlecaps.de/rr/ui) remove all of these brackets